docs/community/maintainer/release.md

```diff
@@ -0,0 +1,62 @@
+# Release Flow
+
+## Overview
+Trivy adopts [conventional commit messages][conventional-commits], and [Release Please][release-please] automatically creates a [release PR](https://github.com/googleapis/release-please?tab=readme-ov-file#whats-a-release-pr) based on the messages of the merged commits.
+This release PR is automatically updated every time a new commit is added to the release branch.
+See the [Release Please documentation][release-please] for more information.
+
+If a commit has the prefix `feat:`, a PR is automatically created to increment the minor version, and if a commit has the prefix `fix:`, a PR is created to increment the patch version.
+When the PR is merged, GitHub Actions automatically creates a version tag and the release is performed.
+For detailed behavior, please refer to [the GitHub Actions configuration][workflows].
+
+!!! note
+    Commits with prefixes like `chore` or `build` are not considered releasable, and no release PR is created.
+    To include such commits in a release, you need to either include commits with `feat` or `fix` prefixes or perform a manual release as described [below](#manual-release).
+
+## Flow
+The release flow consists of the following main steps:
+
+1. Creating the release PR (automatically or manually)
+1. Drafting the release notes
+1. Merging the release PR
+1. Updating the release notes
+
+### Automatic Release PR Creation
+When a releasable commit (a commit with `feat` or `fix` prefix) is merged, a release PR is automatically created.
+The title of the PR will be in the format `release: v${version} [${branch}]` (e.g., `release: v0.51.0 [main]`).
+The format of the PR title is important for identifying the release commit, so it should not be changed.
+
+The `release/vX.Y` release branches are also subject to automatic release PR creation for patch releases.
```

Comment:
> Like today, we basically maintain the main branch. We'll create release/v* and backport patches to the branch only when we find critical bugs.

IIUC we will create `release/vX.Y` branch only for patches with bug fixes.

So we need to use the patch part in the branch name to add the ability to release more than one patch for a minor version.

```suggestion
The `release/vX.Y.Z` release branches are also subject to automatic release PR creation for patch releases.
```

Reply:
Oh... I think I've found a flaw in my logic

I wanted to split all the patches into separate branches.
But in this case, for patch `vX.Y.2` and above, we will skip changes from previous patches (or we will have to add them manually).

for `vX.Y` we will simply add the necessary commits and release patches when they are needed

Thanks for the clarification, now I understand this point 👍

Reply:
Creating a branch for a minor version is straightforward to me. We can backport patches from `main` to `release/vX.Y`, and cut vX.Y.1, vX.Y.2... on the branch. Many OSS projects are doing this, although their branch naming conventions are slightly different, `1.12`, `1.12.x`, `release-1.12`, `release-1.12.x`, `release/1.12`, etc.